templates/bake/Controller/controller.twig

```diff
@@ -22,8 +22,13 @@ declare(strict_types=1);
 
 namespace {{ namespace }}\Controller{{ prefix }};
 
+{% if plugin %}
+use {{ appNamespace }}\Controller\AppController;
```

Comment:
Should we check to see if the plugin has it's own AppController? Right now the plugin command will generate an AppController that will never be used. If plugin app controllers were used when they existed we could handle more situations.

Reply:
I dont think in 4.x days this is useful here anymore.
This was mainly a 2.x thing, I never use plugin app controllers here, do you?

Reply:
I agree that we could check this, but that would be somewhat another improvement maybe on top right? As this is not trivial from the template or code probably to check for possible app controller names here.

Reply:
> I agree that we could check this, but that would be somewhat another improvement maybe on top right?

That's fair, I can take care of adding the check. As right now we're creating classes in the plugin that will go unused.

> As this is not trivial from the template or code probably to check for possible app controller names here.

We know the namespace of the plugin and could do a `class_exists()` check for the plugin' app controller.